spyder/plugins/ipythonconsole/widgets/namespacebrowser.py

```diff
@@ -39,12 +39,20 @@ def get_value(self, name):
         reason_big = _("The variable is too big to be retrieved")
         reason_not_picklable = _("The variable is not picklable")
         reason_dead = _("The kernel is dead")
-        reason_other = _("An unkown error occurred. Check the console because "
-                         "its contents could have been printed there")
+        reason_other = _("An unkown error occurred. Check the console because"
+                         " its contents could have been printed there")
         reason_comm = _("The comm channel is not working")
+        reason_missing_package = _("The required package '{module}' to open "
+                                   "this variable is not installed")
+        reason_missing_package_conda = _("{reason_missing_package}. You can "
+                                         "try to install the missing modules "
+                                         "in the environment you are trying "
+                                         "to run Spyder.")
```

Comment:
```suggestion
        reason_missing_package = _(
            "The '{}' package is required to open this variable and it's not "
            "installed alongside Spyder. To fix this problem, please install "
            "it in the same environment that you use to run Spyder."
        )
```

Improve message and variable name (this problem can happen in any kind of environment, not just conda ones).